vars/skipStage.groovy

```diff
@@ -229,6 +229,7 @@ boolean call(Map config = [:]) {
     }
 
     String target_branch = env.CHANGE_TARGET ? env.CHANGE_TARGET : env.BRANCH_NAME
+    String hw_distro = hwDistroTarget2().join("")
```

Comment:
Ahh.  Disregard my previous comment as you used the `2` variant while I was reviewing.